spec/helpers/anchor/form_builder_spec.rb

```diff
@@ -1,6 +1,99 @@
 require "rails_helper"
 
 RSpec.describe Anchor::FormBuilder, type: :helper do
+  describe "#label" do
+    context "when the field is optional" do
+      it "appends '(Optional)' to the label" do
+        model = FakeUser.new
+        result = helper.anchor_form_with model:, url: false do |form|
+          form.label :language
+        end
+
+        page = Capybara.string(result)
+
+        expect(page).to have_css(
+          "label", text: "Language (Optional)", exact_text: true
```

Comment:
I _think_ this can be:

```suggestion
          "label", exact_text: "Language (Optional)"
```